src/index.js

```diff
@@ -1,6 +1,6 @@
 import importRule from './rules/import';
 
-export default {
+module.exports = {
```

Comment:
I don't think having to add some specific babel plugin is a good choice for a library. And according to the ES6 spec the behaviour in babel 6 is the correct one. What do you think?